consensus/Cargo.toml

```diff
@@ -12,21 +12,23 @@ documentation = "https://docs.rs/commonware-consensus"
 
 [dependencies]
 commonware-cryptography = { workspace = true }
-commonware-macros = { workspace = true }
-commonware-p2p = { workspace = true }
-commonware-runtime = { workspace = true }
-commonware-storage = { workspace = true }
 commonware-utils = { workspace = true }
 bytes = { workspace = true }
-rand = { workspace = true }
-rand_distr = { workspace = true }
-thiserror = { workspace = true }
 futures = { workspace = true }
 tracing = { workspace = true }
 prost = { workspace = true }
+cfg-if = { workspace = true }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+commonware-macros = { workspace = true }
+commonware-p2p = { workspace = true }
+commonware-runtime = { workspace = true }
+commonware-storage = { workspace = true }
 prometheus-client = { workspace = true }
-sha2 = { workspace = true }
```

Comment:
Turns out this direct import was unnecessary (and was not caught by the `dependencies` CI)